doc/source/PreConfiguredSites.rst

```diff
@@ -24,7 +24,7 @@ The following naming conventions are used on all fully-supported (tier 1) sites.
 +----------------------------------+---------------------------------------------------------+-------------------+------------------------------+
 | ``skylab-dev``                   | JEDI/Skylab environment for JEDI, models, EWOK          | ``se``            | ``se-apple-clang-14.0.6``    |
 +----------------------------------+---------------------------------------------------------+-------------------+------------------------------+
-| ``neptune-dev``                  | NEPTUNE standalone environment (with xNRL Python)       | ``ne``            | ``ne-oneapi-2024.2.1``       |
+| ``neptune-dev``                  | NEPTUNE-JEDI standalone environment                     | ``ne``            | ``ne-oneapi-2024.2.1``       |
```

Comment:
The `-JEDI` addition is in anticipation of a follow-up PR that will add `neptune-jedi-env` to the `neptune-dev` template. I will otherwise forget making this documentation update.